src/Applications/GAAS_App/mxd04_nnr.py

```diff
@@ -304,6 +313,14 @@ def speciate(self,aer_x,Verbose=False):
         except:
             pass   # ignore it for systems without nitrates
 
+        # Handle brown carbon
+        # --------------------
+        try:
+            self.sampleFile(aer_x,onlyVars=('BRCEXTTAU',),Verbose=Verbose)
```

Comment:
You may want to update to the latest name for BRC used in GOCART2G which is BREXTTAU.

Reply:
thanks for catching that. 
Can you re-approve.